CHANGELOG.md

```diff
@@ -3,6 +3,83 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.13.0
+
+_Aug 15, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Allow [label editing capabilities on the Tree View](https://mui.com/x/react-tree-view/rich-tree-view/editing/) (#13388) @noraleonte
```

Comment:
```suggestion
- 💫 Allow to [edit the label](https://mui.com/x/react-tree-view/rich-tree-view/editing/) of Tree View's items.
```

From what I can see we don't put the author and the PR link on highlights
If we want to put them it would be nice to discuss and align :+1: 

And "capabilities" seems slightly convoluted here.

Reply:
@noraleonte this on may be worth an inline video